cmdstanpy/stanfit.py

```diff
@@ -1231,20 +1222,25 @@ class CmdStanGQ:
     Container for outputs from CmdStan generate_quantities run.
     """
 
-    def __init__(self, runset: RunSet, mcmc_sample: pd.DataFrame) -> None:
+    def __init__(
+        self,
+        runset: RunSet,
+        mcmc_sample: CmdStanMCMC,
+        logger: logging.Logger = None,
```

Comment:
Should be `Optional[logging.Logger]` to be precise

Reply:
fixed